text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,129 @@
+
+>Due to the promotion of anti-democratic, anti-scientific content and conspiracy theories on the platform X, RWTH Aachen University is going to discontinue its activities on it. The platform is not sufficiently addressing hate speech, hatred against minorities, conspiracy theories, and right-wing extremist opinions – in short, it is not doing enough to combat radicalization. We will continue to provide information on our other social media channels (Instagram, LinkedIn, YouTube, TikTok, Threads and Facebook).
+
+>In recent years, UU's reach on X has decreased compared to other platforms, such as LinkedIn. In addition, we have seen how X, often more than other platforms, struggles with the rise of disinformation, fake accounts and online harassment.
+
+>This is for a number of reasons: primarily it no longer works as an effective method of distributing information to our communities. 
+</details>
+</summary>
+
+Also, six months after leaving X/Twitter, [NPR claims that the effect on their audience has been negligible](https://pluralistic.net/2023/10/14/freedom-of-reach/).
+
+## Design
+
+We have to decide what the social media presence of the Rocq Prover will be. The options are:
+
+- No social media presence: leave X/Twitter and do not create accounts on other platforms. This is the only alternative that does not require calling for volunteers. In all other cases, we need to expand the social media team.
+- Keep the CoqLang account on X/Twitter and rename it to Rocq Prover. Expand the team of people managing the account.
+- Keep the CoqLang account on X/Twitter and create new accounts on other platforms. This requires expanding the team of people managing the accounts even more, as it seems unlikely that the same people will manage all the accounts (even in the case of one account automatically reposting from another, because we need to monitor replies and DMs).
+- Leave X/Twitter and create new accounts on other platforms. This still requires finding volunteers to manage the new accounts.
+
+The volunteers we find will depend on the platforms we target. We can have a predefined list of platforms to target, but we can also be open to suggestions from the community. Furthermore, we cannot create accounts on platforms for which we do not find volunteers.
+
+I propose to target the following platforms: Mastodon, Bluesky, and LinkedIn. I also propose to be open to suggestions from the community, but to decide upfront to leave X/Twitter. Of course, an alternative would be to consider X/Twitter in the same way as other platforms, i.e., to continue using it if and only if we find enough interested volunteers.
```

Comment:
We should evaluate how effective the new platforms are after an initial trial period, perhaps a year.

Reply:
I am not confident that we can have a good measure of effectiveness. We may be able to compare raw metrics such as number of reposts or likes, but even those would not tell us much (not all reposts or likes are worth the same, and on opaque platforms like X/Twitter, you are not even sure how reposts and likes influence the recommendation algorithm).